libs/SmartStore/SmartStoreTests/SFSmartSqlWithExternalStorageTests.m

```diff
@@ -30,6 +30,29 @@
 #import <SalesforceSDKCore/SFJsonUtils.h>
 #import "SFSoupSpec.h"
 
+@interface SFSmartSqlTests ()
```

Comment:
Not the end of the world, but in a scenario like this, why not put responsibility of defining the interface on the parent class, then reference that interface here?